support-frontend/assets/pages/paper-subscription-checkout/helpers/options.js

```diff
@@ -22,6 +22,34 @@ import { getVoucherDays } from 'pages/paper-subscription-checkout/helpers/vouche
 import { getHomeDeliveryDays } from 'pages/paper-subscription-checkout/helpers/homeDeliveryDays';
 import { formatMachineDate } from 'helpers/dateConversions';
 
+export type PaperSubscriptions = 'Everyday' | 'Sixday' | 'Weekend' | 'Sunday' | null;
```

Comment:
This already exists in productOptions.js

Reply:
It sort of does - there's a list of active products. I have created a type for this. Thanks for pointing it out.